SingularityUI/app/assets/_index.mustache

```diff
@@ -27,9 +27,9 @@
             hideNewRequestButton: {{{hideNewRequestButton}}},
             defaultCpus: {{{defaultCpus}}},
             defaultMemory: {{{defaultMemory}}},
-            defaultHealthcheckIntervalSeconds: {{{defaultHealthcheckIntervalSeconds}}},
-            defaultHealthcheckTimeoutSeconds: {{{defaultHealthcheckTimeoutSeconds}}},
-            defaultDeployHealthTimeoutSeconds: {{{defaultDeployHealthTimeoutSeconds}}},
+            defaultHealthcheckIntervalSeconds: 5,
+            defaultHealthcheckTimeoutSeconds: 10,
+            defaultDeployHealthTimeoutSeconds: 120,
```

Comment:
can you un-commit these defaults? i'll roll out the legit fix for this to master